### DISCUSSION TITLE
MSSQL sys.schemas not supported?

### DISCUSSION THREAD

**Discussion by norton64** [dbeaver/dbeaver#11146] [Q&A]
Hi There:

I'm using MSSQL on a VM with the regular MSSQL driver, and Azure SQL with the specific Azure driver.  When I try to change, or just browse schemas, I get the the error: `Reference to database and/or server name in 'DATABASE.sys.schemas' is not supported in this version of SQL Server`

I know there's a difference in the Azure hosted SQL flavors that affect the way a sql editor connects to a single database only, but the browser in SSMS still works.  Is this a limitation in the browser of DBeaver with Azure connections?  Have I missed a setting somewhere?

I'm using MSSQL 2016 and 2019 with DBeaver Community 7.3.3

Cheers and thanks,
Simon

**Comment by jankrejcik** [comment]
https://github.com/dbeaver/dbeaver/issues/5258
